spyder/app/mainwindow.py

```diff
@@ -1374,7 +1374,7 @@ def post_visible_setup(self):
         if self.splash is not None:
             self.splash.hide()
 
-        if self.open_project:
+        if self.open_project and not self.first_spyder_run:

```

Comment:
```suggestion
        if (self.open_project and
                (running_in_mac_app() and not self.first_spyder_run)):
```

Just to avoid this condition only when running in Mac app.